Rewrite/src/Rewrite.CSharp/CSharpPrinter.cs

```diff
@@ -493,6 +509,38 @@ protected void VisitStatement(JRightPadded<Statement> paddedStat, CsRightPadded.
         _delegate.PrintStatementTerminator(paddedStat.Element, p);
     }
 
+    public override J? VisitTypeParameterConstraintClause(Cs.TypeParameterConstraintClause typeParameterConstraintClause, PrintOutputCapture<TState> p)
+    {
+        BeforeSyntax(typeParameterConstraintClause, CsSpace.Location.TYPE_PARAMETERS_CONSTRAINT_CLAUSE_PREFIX, p);
+        p.Append("where");
+        VisitRightPadded(typeParameterConstraintClause.Padding.TypeParameter, CsRightPadded.Location.TYPE_PARAMETER_CONSTRAINT_CLAUSE_TYPE_PARAMETER,  p);
+        p.Append(":");
+        VisitContainer("", typeParameterConstraintClause.Padding.TypeParameterConstraints, CsContainer.Location.TYPE_PARAMETER_CONSTRAINT_CLAUSE_TYPE_CONSTRAINTS, ",", "", p);
```

Comment:
When a container is always printed without prefix and suffix, you might as well directly use List<JRightPadded>.

Reply:
Can't use list because you still need to account for whitespace between commas and all elements need to be `JRightPadded` as a result. It just becomes easier to wrap it as `JContainer` at that point as we already have many helper methods for working with it and it properly handles separated syntax.

Reply:
For List<JRightPadded> there are also helper methods, so there isn't much of a difference there. Or what specifically do you miss from that?